bsp/src/mill/bsp/ModuleUtils.scala

```diff
@@ -162,7 +162,7 @@ object ModuleUtils {
     val binarySource =
       if (sources) all.filter(url => isPathSourceJar(url))
       else all.filter(url => !isPathSourceJar(url))
-    binarySource.filter(path => exists(path)).map(_.wrapped.toString)
+    binarySource.filter(path => exists(path)).map(_.toNIO.toUri.toString)
```

Comment:
I'm wondering why we don't use a better return type here. At least, we should add some Scaladoc to line 126 to make it clear, that the returned `String`s are meant to be URIs.